src/main/java/io/chazza/advancementapi/AdvancementAPI.java

```diff
@@ -238,14 +252,13 @@ public String toString() {
         }
     }
 
-
     //BEGIN CLASSES
     @Builder(builderMethodName = "hiddenbuilder")
     public static class Condition {
         protected String name;
-        protected JSONObject set;
+        protected JsonObject set;
```

Comment:
A condition has always a name and a JsonObject.

The trigger is defined in class Trigger, should be named Criteria instead.


![bildschirmfoto 2017-06-21 um 10 21 23](https://user-images.githubusercontent.com/1420893/27374256-74b2f498-566b-11e7-8e07-aec2a87d96b9.png)